README.rst

```diff
@@ -40,31 +40,27 @@ Next step is to install assets (js, css etc.):
     plupload
     ...
 
-Now you can follow the standard invenio installation and development procedures, such as running `inveniomanage collect`.
+Now you can follow the standard Invenio installation and development procedures, such as running `inveniomanage collect`.
 
 ---------------------
 Demosite installation
 ---------------------
-As INSPIRE uses `Mixer <https://github.com/klen/mixer>`_ to load database fixtures, you should install the library:
+Since the database was already initialized while installing Invenio, you run:
 
 .. code-block:: bash
 
-     $ pip install -e .[mixer]
+     $ inveniomanage database recreate
```

Comment:
Indeed `create` is part of those instructions which probably even can be removed from here, or turned into a "tip" on how to reload the database and load demo records.